src/Content/Text/BBCode.php

```diff
@@ -451,7 +451,7 @@ private static function proxyUrl($image, $simplehtml = self::INTERNAL, $uriid =
 		// Only send proxied pictures to API and for internal display
 		if (!in_array($simplehtml, [self::INTERNAL, self::API])) {
 			return $image;
-		} elseif ($uriid) {
+		} elseif ($uriid > 0) {
```

Comment:
Why did we need to be more specific?

Reply:
When rendering the preview we set the `uri-id` of the item array to `-1`.

Reply:
See here: https://github.com/friendica/friendica/blob/develop/mod/item.php#L676-L682